uint8 flag
float32 vx
float32 vy
float32 vz
float32 yaw
---
bool result 
